druid/src/window.rs

```diff
@@ -426,7 +426,9 @@ impl<T: Data> Window<T> {
         for &r in invalid.rects() {
             piet.clear(
                 Some(r),
-                if self.transparent {
+                // we are maintaining backwards compatibility here where setting
+                // transparent to true explicitly would make a fully transparent window.
+                if self.transparent && env.get(crate::theme::WINDOW_BACKGROUND_COLOR).as_rgba().3 == 255.0 {
```

Comment:
first, this check is wrong, you need `.as_rgba8().3 == 255` (range of rgba() is 0.-1.)

after fixing the check, this makes all default windows fully transparent (because default window background has alpha 255).

I think it is ok backwards compatibility because `transparent` is not yet released to crates.io (only on git)

Reply:
ah, yeah if we don't care about backwards compat then good, makes it less messy.

Reply:
after playing with this logic today. it still requires people to set two flags, but instead of for transparent its now for opaque.

Reply:
and it breaks a bunch of tests =p

Reply:
and if its fine to break backwards compat rather just remove this method entirely and go with replacing the boolean with a color we check in the backend.

Reply:
it is a good idea to handle background color in the backend. it might help when ui doesn't draw fast enough to keep up with resize.